test/lib/ufe/testClipboard.py

```diff
@@ -52,6 +52,9 @@ def setUp(self):
     def testClipboardCopyPaste(self):
         '''Basic test for the Clipboard copy/paste support.'''
 
+        # Verify that the method to set the clipboard format exists.
+        self.assertTrue(callable(getattr(mayaUsd.ufe, 'setClipboardFileFormat', None)))
```

Comment:
Does this have to run for Maya 2024 and earlier?

Reply:
This entire test file is only included when the Ufe version being built with contains the clipboard support. So this file won't be included for 2024.